webpack/modules/fs.js

```diff
@@ -1,3 +1,4 @@
 const memfs = require("memfs");
 
 module.exports = { ...memfs };
+globalThis.fs = memfs;
```

Comment:
do you need this anymore? is this used inside go-wasm.js?

Reply:
Yes, still need this. The go-wasm.js doesn't require fs, it looks for globalThis.fs and if it is undefined it creates it's own globalThis.fs. I opted to add it here rather than trying to replace the checks for globalThis.fs. I think if I do the other changes you suggested so we don't need the prebuild plugin then this way would be the only way to get the correct fs patch for the go-wasm.js file.

Reply:
since this bursts the bubble that webpack creates around our UMD module, after I write tests I'll try to see if I can somehow get it to work properly now that `go-wasm.js` is actually passing through webpack.